src/modules/esl-anchornav/core/esl-anchor.less

```diff
@@ -0,0 +1,4 @@
+[esl-anchor] {
```

Comment:
Just a small extra question (not for MVP)

Can we additionally control the tolerance of anchor tracking?

Reply:
Yes, will do it next wave